library/worldgen/biome/src/main/java/org/quiltmc/qsl/worldgen/biome/impl/NetherBiomeData.java

```diff
@@ -66,10 +75,40 @@ public static boolean canGenerateInNether(RegistryKey<Biome> biome) {
 			}
 		}
 
-		return NETHER_BIOMES.contains(biome);
+		return NETHER_BIOMES.contains(biome) || NETHER_BIOME_NOISE_POINTS.containsKey(biome);
 	}
 
 	private static void clearBiomeSourceCache() {
 		NETHER_BIOMES.clear(); // Clear cached biome source data
 	}
+
+	private static MultiNoiseUtil.ParameterRangeList<Holder<Biome>> withModdedBiomePoints(MultiNoiseUtil.ParameterRangeList<Holder<Biome>> defaultEntries, Registry<Biome> biomeRegistry) {
+		if (NETHER_BIOME_NOISE_POINTS.isEmpty()) {
+			return defaultEntries;
+		}
+
+		List<Pair<MultiNoiseUtil.NoiseHypercube, Holder<Biome>>> entries = new ArrayList<>(defaultEntries.getEntries());
+
+		for (Map.Entry<RegistryKey<Biome>, MultiNoiseUtil.NoiseHypercube> entry : NETHER_BIOME_NOISE_POINTS.entrySet()) {
+			if (biomeRegistry.contains(entry.getKey())) {
+				entries.add(Pair.of(entry.getValue(), biomeRegistry.getHolderOrThrow(entry.getKey())));
```

Comment:
```suggestion
				entries.add(Pair.of(entry.getValue(), biomeRegistry.getHolderOrThrow(entry.getKey())));
```

The old code was using `method_44298` (should be named `getOrCreateHolderOrThrow`), should we use the same?

Reply:
The "create" bit has caused trouble when it was still on the old mixin because that meant that the value was never loaded in; However, now that it's being applied late, it shouldn't do much harm (and testing it, I can confirm that)